packages/contracts/src/networks.ts

```diff
@@ -146,7 +146,7 @@ export const SPHINX_NETWORKS: Array<SupportedNetwork> = [
       envKey: 'POLYGON_ETHERSCAN_API_KEY',
     },
     currency: 'MATIC',
-    dripSize: '1',
+    dripSize: '0.15',
```

Comment:
I accidentally updated this such that it no longer matches the real on-chain value which causes problems.